lib/index.js

```diff
@@ -178,10 +185,10 @@ Socket.prototype._receive = function () {
             return;
         }
     }
-    if (msg == -1) return;
+  if (msg == -1) return '';
```

Comment:
if you return nothing streams will blow up with an uncaught exception on the process. and especially so if the returned msg is ever `-1`. actually any number type for that matter. You can only give streams a string or a buffer basically.

Anyway in practice this only crops up for req/rep and surveyor/respondent scenarios IMO